buildSrc/src/main/kotlin/DocsLandingPage.kt

```diff
@@ -0,0 +1,53 @@
+import org.gradle.api.Project
+import software.amazon.smithy.utils.CodeWriter
+import java.io.File
+
+/**
+ * Generate a basic docs landing page into [outputDir]
+ *
+ * The generated docs will include links to crates.io, docs.rs and GitHub examples for all generated services. The generated docs will
+ * be written to `docs.md` in the provided [outputDir].
+ */
+fun Project.docsLandingPage(awsServices: List<AwsService>, outputDir: File) {
```

Comment:
Are the services always ordered deterministically when passed into this function?

Reply:
they are, but I just reorded them by human name to be double sure